Caltech-IPAC/firefly
# DM-8985: server now finding image with mask extension, mask UI implemented

DM-8985: server now finding image with mask extension

 - server discovers mask layers in fits file
 - mask and other overlay information returned as addtional data for image load
 - give user options to load mask in drawing layers panel
 - add show all and hide all buttons
 - improvement in consistent color choosing of mask layers
 - mask can be lazy loaded
 - enable all mask in group, on/off in group,
 - fixed FileInfo null retured bug in last pull request
 - rotate ui: matching group, matching last plot
 - fixed zoom locking now consistently working, added LEVEL as plot zoom options to disable locking
 - made the default ZoomType TO_WIDTH_HEIGHT
 - fixed small toolbar layout background size issue

**To Test**:
 - use validation_data_hsc repository:
 - look in directory validation_data_hsc/DATA/00533/HSC-R/corr
 - load any CORR*.fits
    interesting mask:
           - bit number 4, 5,  and 11 are interesting to turn on.
           - CORR-0903342-100.fits has bad pixels so try bit number 1 as well
 - bring up layer control
 - enable the mask layers, the layers should appear
 - turn any on and off 
- try the show all or hide all buttons
- try zoom
- try rotate and flip though they are a little flaky

**known bugs**:
 - flip does not work if you flip before you turn on a mask bit
 - flip and rotate are a fragile in general, might need to look into it more deeply in another ticket
